### PR TITLE
Initial Project Structure

### PR DESCRIPTION
**Pull request by meevee98**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        #   4 - Beta
+        #   5 - Production/Stable
+        'Development Status :: 3 - Alpha',
+
+        # Indicate who your project is intended for
+        'Intended Audience :: Developers',
+        'Topic :: Software Development :: Build Tools',
+
+        # Pick your license as you wish (should match "license" above)
+        'License :: OSI Approved :: MIT License',
```
Comment (lock9) [inline setup.py:45]: I think we are moving to Apache 2. @lllwvlvwlll @ixje  can you confirm?

Comment (lock9) [inline setup.py:45]: I think we are moving to Apache 2. @lllwvlvwlll @ixje  can you confirm?

Comment (lock9) [reply setup.py:45]: It is apache, I just confirmed

**Comment by lllwvlvwlll** [comment]
I've added the Circleci scaffold.  As a general rule, we should try to maintain most of the operations in the Makefile, then call it within our CI workflows so we can manually perform the operations if needed.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+
+setup(
+    name='neo3-boa',
+
+    # Versions should comply with PEP440.  For a discussion on single-sourcing
+    # the version across setup.py and the project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+
+    description='A Python compiler for the Neo Virtual Machine',
```
Comment (ixje) [inline setup.py:21]:
```suggestion
    description='A Python compiler for the Neo3 Virtual Machine',
```
I missed this one

**Comment by ixje** [comment]
Thanks for the setup !

**Pull request merged**